lib/ruby_lsp/executor.rb

```diff
@@ -97,7 +97,7 @@ def run(request)
         # Run listeners for the document
         emitter = EventEmitter.new
         document_symbol = Requests::DocumentSymbol.new(emitter, @message_queue)
-        document_link = Requests::DocumentLink.new(uri, emitter, @message_queue)
+        document_link = Requests::DocumentLink.new(uri, emitter, @message_queue, document.comments)
```

Comment:
We've been putting extra arguments other than `emitter` and `message_queue` first. Can we put this after `uri` please?